Automattic/jetpack
# Additional CSS: Update the upgrade nudge with the correct plan

Fixes https://github.com/Automattic/dotcom-forge/issues/9519

Update the upgrade nudge in Additional CSS to display the correct plan name. 

## Proposed changes:

* Update the upgrade nudges for the Additional CSS to display "Personal/Premium" instead of "Explorer"
* On Atomic, it should say "Business" instead of "Creator"

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On Simple Sites, use the testing command from the comments to update your sandbox
* On your sandbox, enable the GS flag by adding `const GLOBAL_STYLES_ON_PERSONAL_PLAN = true;` in 0-sandbox.php 
* Go to Customizer > Additional CSS
* Notice that the Plan in the upgrade notice is "Personal". 
* Disable the flag and now it should say "Premium" instead of Explorer.
